doc/userguide/performance/high-performance-config.rst

```diff
@@ -1,19 +1,310 @@
 High Performance Configuration
 ==============================
 
-If you have enough RAM, consider the following options in suricata.yaml to off-load as much work from the CPU's as possible:
+NIC
+---
+
+One of the major dependencies for Suricata's performance is the Network Interface Card. There are many vendors and possibilities. Some NICs have and require their own specific instructions of how to set up the NIC and utilize all of its benefits properly in order to run Suricata  - like Napatech, Netronome, Accolade, Myricom etc.
```

Comment:
Long sentences like this are supposed to be cut at ~80 chars. It will not change the output in pdf or html, but will make it more readable in editors.